### DISCUSSION TITLE
Remove animation on new tab

### DISCUSSION THREAD

**Discussion by j-freddy** [bluecaret/carettab#346] [Help and Support] [answered]
When a new tab is opened, there is a short [fade-in from white] animation. How can I remove this animation? If this is not possible, may this feature be considered?

**Comment by bluecaret** [comment, marked as answer]
I can look into making that an option, though the way the extension works it might be feasible. I had based the adding of the animation based on how similar extensions handle the loading of the tab. But maybe I can revisit having it at all. The thought process was it would be better than a flash. The initial white (or black if in dark mode) in unavoidable, so by adding the animation it would feel less intense. Thank you!

**Reply by j-freddy** [reply]
Thank you for the consideration.

Compare CaretTab to Tabby Cat @ https://tabbycats.club/, which displays a new tab without any animation. Up until you pointed it out, I had never noticed the white flash before the tab loads.

My personal UX preference has always been practicality so I like to configure my utility/productivity apps to be quick & snappy :)
